geoportal/geoportailv3_geoportal/static-ngeo/js/catalog/CatalogController.js

```diff
@@ -119,9 +121,14 @@ const exports = function($scope, appThemes, appTheme,
           mixed: true,
           theme: this.appTheme_.getCurrentTheme()
         })
-      } else {
+        break;
```

Comment:
@RemiDesgrange This beaks the build "Module build failed: SyntaxError: Unsyntactic break (124:8)"

Reply:
Ok problème de rebase. Je regarde de suite.

Reply:
Fixed in #2650